apps/workflow-cf.yml.j2

```
@@ -67,11 +67,13 @@ Resources:
           {% for command in task['command'] %}
           - {{ command }}
           {% endfor %}
+        {% if task.get('secrets', []) %}
```

I don't think we need to specify `[]` as the default value here, given that `None` and `[]` are both falsy.
```suggestion
        {% if task.get('secrets') %}
```